subsys/usb/device_next/class/bt_hci.c

```diff
@@ -326,14 +434,34 @@ static int bt_hci_acl_out_cb(struct usbd_class_data *const c_data,
 	}
 
 	if (hci_data->acl_buf == NULL) {
-		hci_data->acl_buf = bt_buf_get_tx(BT_BUF_ACL_OUT, K_FOREVER,
+		uint8_t type = BT_HCI_H4_ACL;
+
+		// FIXME: remove?
+		LOG_HEXDUMP_DBG(buf->data, buf->len, "ACL data received from USB");
+
+		struct bt_hci_iso_hdr *hdr = (struct bt_hci_iso_hdr *)buf->data;
+
+		LOG_DBG("ACL data received, len %zu, handle %u", buf->len, hdr->handle);
+
+		uint16_t handle = bt_acl_handle(sys_le16_to_cpu(hdr->handle));
+
+		// TODO: snoop ISO handles from ISO commands
+		// this would allow to use controllers other than ll_sw
+		if (IS_CIS_HANDLE(handle) || 
+			IS_ADV_ISO_HANDLE(handle) ||
+		    IS_SYNC_ISO_HANDLE(handle)) {
+			LOG_DBG("CIS/ISO handle %u, type %u", hdr->handle, type);
+			type = BT_HCI_H4_ISO;
+		}
```

Comment:
Basically, I'm talking about this: https://github.com/torvalds/linux/blob/89be9a83ccf1f88522317ce02f854f30d6115c41/net/bluetooth/hci_core.c#L2939

We should keep track of CIS/BIS/other ISO setup/destroy commands host issues, and this would allow us to know when an ACL packet host sends us is an ISO packet.